src/hooks/useLegacyUIURL/index.ts

```diff
@@ -45,3 +40,19 @@ export const useLegacyUIURL = (): string | null => {
 
   return legacyUIUrl;
 };
+
+/**
+ * `slugToId` is a helper function that takes a Params object from react-router-dom's matchPath and returns a slug value from the slugs object if it is a suitable id.
+ * @param matchedPathParams - Params object from react-router-dom's matchPath
+ * @returns string | undefined
+ */
+const slugToId = (matchedPathParams: Params): string | undefined => {
```

Comment:
Good catch on this I updated it so that non id slugs are no longer included in the check. I preferred this approach over the previous implementation of just casing on every single slug.